src/components/shared/formItems/ProjectBondingCurveRate.tsx

```diff
@@ -44,7 +44,7 @@ function BondingCurveRateExtra({ disabled }: { disabled?: boolean }) {
       {disabled && (
         <FormItemWarningText>
           <Trans>
-            Disabled when your project's distribution limit is{' '}
+            Disabled when your funding cycle's distribution limit is{' '}
```

Comment:
Don't want to step on anyone's toes here, but just some suggestions. @mejango WDYT of removing possessional language (where makes sense). So instead here, saying:
> Disabled when **the** funding cycle's distribution limit is...